test/Directory.Packages.props

```diff
@@ -0,0 +1,47 @@
+<Project>
+  <Import Project="..\Directory.Packages.props" />
```

Comment:
I think that it will be good idea to reference here src\Directory.Packages.props
but without the section related to  CommonExcludedAssets.props.
(Mainly to have OpenTelemetry* defined only in one place),

Reply:
I thought about that as well, but there are only four dependencies that are common:

```
# cat src/Directory.Packages.props test/Directory.Packages.props | sort | uniq -d | grep Version
    <PackageVersion Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.4.0-rc.2" />
    <PackageVersion Include="OpenTelemetry.Instrumentation.Http" Version="1.0.0-rc9.11" />
    <PackageVersion Include="OpenTelemetry.Instrumentation.Wcf" Version="1.0.0-rc.8" />
    <PackageVersion Include="OpenTelemetry" Version="1.4.0-rc.2" />
```

I've put them into `Directory.Packages.props` file in the root folder.